### PR TITLE
feat: Home page components

### PR DESCRIPTION
**Pull request by erenfn**

### CODE REVIEW

File: frontend/src/components/HomePageComponents/CreateActivityButton/CreateActivityButton.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const icon = placeholder === 'Create a welcome tour' 
+    ? <DirectionsBusFilledOutlinedIcon style={{ color: '#667085', fontSize:'2rem' }} /> 
+    : <WbIncandescentOutlinedIcon style={{ color: '#667085', fontSize:'2rem' }} />;
+
+  return (
+    <Button
+      variant="contained"
+      startIcon={icon}
+      onClick={onButtonClick}
+      sx={{
```
Comment (uparkalau) [inline frontend/src/components/HomePageComponents/CreateActivityButton/CreateActivityButton.jsx:18]: separate it into theme file or css, use contants for colors.

File: frontend/src/components/HomePageComponents/DateDisplay/DateDisplay.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const currentDateFormatted = currentDate.toLocaleDateString(undefined, options);
+
+  return (
+    <div className={styles.date}>
+      Today is {currentDayOfWeek}, {currentDateFormatted}
+    </div>
+  );
+};
+
+// DateDisplay.propTypes = {
```
Comment (uparkalau) [inline frontend/src/components/HomePageComponents/DateDisplay/DateDisplay.jsx:22]: delete if do not need it. But better to use props type

File: frontend/src/components/HomePageComponents/StatisticCards/StatisticCards.module.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.statisticCard {
+    border: 1px solid #EAECF0;
```
Comment (uparkalau) [inline frontend/src/components/HomePageComponents/StatisticCards/StatisticCards.module.scss:2]: use constants for the color definition

File: frontend/src/scenes/dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            <div className={styles.container}>
+                <div className={styles.top}>
+                    <UserTitle userName={username}/>
+                    <DateDisplay/>
+                </div>
+                <div className={styles.text}>
+                Start with a popular onboarding process
+                </div>
+                <div className={styles.activityButtons}>
+                    <CreateActivityButton
```
Comment (uparkalau) [inline frontend/src/scenes/dashboard/Dashboard.jsx:24]: create a list component for this repetitions

File: frontend/src/scenes/dashboard/Dashboard.jsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                    />
+                    <CreateActivityButton 
+                        placeholder="Add a hint to your app" 
+                    />
+                    <CreateActivityButton 
+                        placeholder="Create a new banner" 
+                    />
+                </div>
+                <div className={styles.statisticCards}>
+                    <StatisticCard
```
Comment (uparkalau) [inline frontend/src/scenes/dashboard/Dashboard.jsx:35]: Create a static cards list component

**Review by uparkalau — CHANGES_REQUESTED** [review]
Jus a little bit of refactoring and then you can merge it

File: frontend/src/components/HomePageComponents/CreateActivityButton/CreateActivityButton.jsx (hunk 1 above)
Comment (erenfn) [reply frontend/src/components/HomePageComponents/CreateActivityButton/CreateActivityButton.jsx:18]: MUIButton is kinda annoying in terms of applying css. The easiest way to apply styles is by using sx. Only the button needs some css. Can't I just leave it like this?

**Review by uparkalau — APPROVED** [review]
Looks good you can merge it.

**Pull request merged**